apps/frontend/src/app/pages/person/user-profile/components/education/education.component.html

```diff
@@ -0,0 +1,154 @@
+              >
+              </wagademy-input>
+            </wagademy-form-field>
+          </div>
+          <div class="mb-4">
+            <wagademy-form-field
+              [control]="educationItem.get('fieldOfStudy')!"
+              label="Field of study"
+            >
+              <wagademy-input
+                placeholder="Course name"
+                type="text"
+                formControlName="fieldOfStudy"
+              >
+              </wagademy-input>
+            </wagademy-form-field>
+          </div>
+          <div class="form-control mb-4 w-40">
+            <label class="label cursor-pointer">
+              <input type="checkbox" class="checkbox checkbox-primary" />
+              <span class="label-text text-neutral-950"
+                >I'm still studying</span
+              >
+            </label>
+          </div>
+          <div class="flex flex-col lg:flex-row lg:gap-10">
+            <div class="lg:w-1/2 mb-4">
+              <wagademy-form-field
+                [control]="educationItem.get('startDate')!"
+                label="Start"
+              >
+                <wagademy-input
+                  placeholder="MM / DD / YYYY"
+                  type="date"
+                  formControlName="startDate"
+                >
+                </wagademy-input>
+              </wagademy-form-field>
+            </div>
+            <div class="lg:w-1/2 mb-4">
+              <wagademy-form-field
+                [control]="educationItem.get('endDate')!"
+                label="End"
+              >
+                <wagademy-input
+                  placeholder="MM / DD / YYYY"
+                  type="date"
+                  formControlName="endDate"
+                >
+                </wagademy-input>
+              </wagademy-form-field>
+            </div>
+          </div>
+          <div>
+            <wagademy-form-field
+              label="About"
+              [control]="educationItem.get('description')!"
+            >
+              <wagademy-text-area
+                placeholder="Tell your description"
+                formControlName="description"
+              ></wagademy-text-area>
+            </wagademy-form-field>
+          </div>
+          @if (i > 0) {
+          <button
+            class="text-neutral-950 font-bold flex items-center gap-1 disabled:text-neutral-300"
+            (click)="removeEducationItem(i)"
+          >
+            Remove
+            <img
+              src="./assets/icons/icon-trash.svg"
+              width="14px"
+              height="14px"
+              alt="trash icon"
+              fetchpriority="auto"
+            /></button
+          >}
+        </div>
+      </div>
+    </form>
+    <div
+      class="flex flex-col sm:flex-row sm:justify-end items-center mt-14 mb-8 gap-4"
+    >
+      <button
+        class="btn btn-primary text-base-100 text-xl w-48"
+        (click)="previousStep.emit()"
+      >
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/90660988/4592d201-cbb1-4e56-b95f-84ae24bc7de1)
Your button is different from figma